Makefile.am

```diff
@@ -147,6 +149,7 @@ libmesos_tests_la_CPPFLAGS =				\
   -I$(MESOS_BUILD_DIR)/include/mesos			\
   -I$(ZOOKEEPER)/include				\
   -I$(ZOOKEEPER)/generated				\
+  -I$(NVML)						\
```

Comment:
One of the test files (`mock_docker.hpp`) will include a file that will include the NVML header.  Depending on how you've configured, the build system may or may not find the rebundled header.